### PR TITLE
remove compat annotations applicable only for pre-LTS

### PR DESCRIPTION
**Pull request by KristofferC**
These are starting to clutter the documentation quite heavily. For people stuck on very old Julia versions, there are always the older builds of the manual.

### CODE REVIEW

File: stdlib/LinearAlgebra/src/blas.jl [hunk 1]
```diff
@@ -988,10 +982,7 @@ The scalar inputs `α` and `β` must be complex or real numbers.
 
 The array inputs `x`, `y` and `AP` must all be of `ComplexF32` or `ComplexF64` type.
 
-Return the updated `y`.
-
-!!! compat "Julia 1.5"
-    `hpmv!` requires at least Julia 1.5.
+Return the updated `y`..
```
Comment (giordano) [inline stdlib/LinearAlgebra/src/blas.jl:985]:
```suggestion
Return the updated `y`.
```

File: stdlib/LinearAlgebra/src/factorization.jl [hunk 2]
```diff
@@ -24,10 +24,7 @@ checknonsingular(info) = checknonsingular(info, RowMaximum())
 """
     issuccess(F::Factorization)
 
-Test that a factorization of a matrix succeeded.
-
-!!! compat "Julia 1.6"
-    `issuccess(::CholeskyPivoted)` requires Julia 1.6 or later.
+Test that a factorization of a matrix succeeded..
```
Comment (giordano) [inline stdlib/LinearAlgebra/src/factorization.jl:27]:
```suggestion
Test that a factorization of a matrix succeeded.
```

File: base/sysinfo.jl [hunk 3]
```diff
@@ -385,10 +376,7 @@ See documentation in [Handling Operating System Variation](@ref).
 
 !!! note
     Not to be confused with `Sys.isbsd()`, which is `true` on DragonFly but also on
-    other BSD-based systems. `Sys.isdragonfly()` refers only to DragonFly.
-!!! compat "Julia 1.1"
-    This function requires at least Julia 1.1.
-"""
+    other BSD-based systems. `Sys.isdragonfly()` refers only to DragonFly."""
```
Comment (giordano) [inline base/sysinfo.jl:379]: To minimise the diff
```suggestion
    other BSD-based systems. `Sys.isdragonfly()` refers only to DragonFly.
"""
```

File: base/sysinfo.jl [hunk 4]
```diff
@@ -371,10 +365,7 @@ See documentation in [Handling Operating System Variation](@ref).
 
 !!! note
     Not to be confused with `Sys.isbsd()`, which is `true` on NetBSD but also on
-    other BSD-based systems. `Sys.isnetbsd()` refers only to NetBSD.
-!!! compat "Julia 1.1"
-    This function requires at least Julia 1.1.
-"""
+    other BSD-based systems. `Sys.isnetbsd()` refers only to NetBSD."""
```
Comment (giordano) [inline base/sysinfo.jl:368]:
```suggestion
    other BSD-based systems. `Sys.isnetbsd()` refers only to NetBSD.
"""
```

File: base/sysinfo.jl [hunk 5]
```diff
@@ -357,10 +354,7 @@ See documentation in [Handling Operating System Variation](@ref).
 
 !!! note
     Not to be confused with `Sys.isbsd()`, which is `true` on OpenBSD but also on
-    other BSD-based systems. `Sys.isopenbsd()` refers only to OpenBSD.
-!!! compat "Julia 1.1"
-    This function requires at least Julia 1.1.
-"""
+    other BSD-based systems. `Sys.isopenbsd()` refers only to OpenBSD."""
```
Comment (giordano) [inline base/sysinfo.jl:357]:
```suggestion
    other BSD-based systems. `Sys.isopenbsd()` refers only to OpenBSD.
"""
```

File: base/sysinfo.jl [hunk 6]
```diff
@@ -343,10 +343,7 @@ See documentation in [Handling Operating System Variation](@ref).
 
 !!! note
     Not to be confused with `Sys.isbsd()`, which is `true` on FreeBSD but also on
-    other BSD-based systems. `Sys.isfreebsd()` refers only to FreeBSD.
-!!! compat "Julia 1.1"
-    This function requires at least Julia 1.1.
-"""
+    other BSD-based systems. `Sys.isfreebsd()` refers only to FreeBSD."""
```
Comment (giordano) [inline base/sysinfo.jl:346]:
```suggestion
    other BSD-based systems. `Sys.isfreebsd()` refers only to FreeBSD.
"""
```

**Comment by vtjnash** [comment]
Would it be possible to change the rendering so that it is just a tiny badge? I usually appreciate these less for their value in documenting the current version, but for their value in seeing that I may need to drop support for an older version when switching to a newer API.

**Comment by vtjnash** [comment]
Looks significantly conflicted now

**Pull request closed without merging**